### PR TITLE
Local kind install

### PR DESCRIPTION
**Pull request by johnsmartco**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/install/local/_index.md [hunk 1]
```diff
@@ -1,441 +1,2316 @@
...
+{{% /alert %}}
+
+### Verify cluster status
+
+On one of the data plane clusters, verify the cluster status. Example:
+
+```bash
+kubectx kind-k8ssandra-1
+
+$ kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -- /bin/bash
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:1299]:
```suggestion
kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- nodetool -u demo-superuser -pw 3CAvGWc4mRna8tODJgeN status
```

Just had the wrong command here, switching over to this will get the nodetool status command run automatically.

I also added the `-c cassandra` to force the container -- I realize that part was omitted from own previous docs, it's not totally required since it'll be the first container, but this will force it and make sure you don't have to updating this later if that should ever change in terms of default ordering.

File: docs/content/en/install/local/_index.md [hunk 2]
```diff
@@ -1,441 +1,2316 @@
...
+```
+
+**Output plus nodetool example:**
+
+```bash
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:1304]: The change above will change this output to something a bit tidier like this:

```
Datacenter: dc1
===============
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address      Load       Tokens       Owns (effective)  Host ID                               Rack
UN  172.18.0.10  327.02 KiB  256          100.0%            9e3e48ee-529e-4b2a-9bf2-39575d32f578  default
UN  172.18.0.11  338.79 KiB  256          100.0%            305616ce-3440-4d37-b9be-32bca624abb7  default
UN  172.18.0.8   304.01 KiB  256          100.0%            0a0864b7-968a-4e07-839f-a5abb2e3d0a4  default
Datacenter: dc2
===============
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address      Load       Tokens       Owns (effective)  Host ID                               Rack
UN  172.18.0.16  355.6 KiB  256          100.0%            f2bd31ef-5ca5-4c28-a9b2-bac28f76af4f  default
UN  172.18.0.15  368.66 KiB  256          100.0%            878e519b-a6f4-4aff-b8ab-c1fb30679847  default
UN  172.18.0.13  344.76 KiB  256          100.0%            3efa6a2f-47d1-49e3-ba93-0a58870e7c0f  default
Datacenter: dc3
===============
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address      Load       Tokens       Owns (effective)  Host ID                               Rack
UN  172.18.0.18  389.07 KiB  256          100.0%            9fc9983e-24a6-415d-9a4d-10d3deff4ca4  default
UN  172.18.0.19  376.66 KiB  256          100.0%            e36d3a62-b521-4a50-b86a-2b392c972195  default
UN  172.18.0.20  401.81 KiB  256          100.0%            396ebb0a-da34-47c9-8f74-0816b5658f86  default
```

File: docs/content/en/install/local/_index.md [hunk 3]
```diff
@@ -1,441 +1,2316 @@
...
+echo $CASS_PASSWORD
+```
 
+**Output example - your value will be different:**
+```bash
+ACK7dO9qpsghIme-wvfI
+```
 {{% alert title="Tip" color="success" %}}
-Alternatively, you can download the individual charts directly from the project's [releases](https://github.com/k8ssandra/k8ssandra/releases) page.
+You'll use the extract credentials for subsequent authentication in deployed containers.
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:558]:
```suggestion
You'll use the extracted credentials for subsequent authentication in deployed containers.
```

File: docs/content/en/install/local/_index.md [hunk 4]
```diff
@@ -1,441 +1,2316 @@
...
-        adminUser: admin
-        adminPassword: admin123
-    stargate:
-      enabled: true
-      replicas: 1
-      heapMB: 256
-      cpuReqMillicores: 200
-      cpuLimMillicores: 1000
-    ```
+### Test a few operations
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:586]: I'm not sure what you'd prefer here, but in the Azure install issue I streamlined my examples a bit to be more copy and paste friendly (after I got smart and had to do it for the 20th time 😉 

Take a look at https://github.com/k8ssandra/k8ssandra/issues/1281

I use a series of commands like:

```
% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "CREATE KEYSPACE test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "CREATE TABLE test.users (email text primary key, name text, state text);"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "insert into test.users (email, name, state) values ('john@gamil.com', 'John Smith', 'NC');"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "insert into test.users (email, name, state) values ('joe@gamil.com', 'Joe Jones', 'VA');"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "insert into test.users (email, name, state) values ('sue@help.com', 'Sue Sas', 'CA');"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "insert into test.users (email, name, state) values ('tom@yes.com', 'Tom and Jerry', 'NV');"

% kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- cqlsh -u $CASS_USERNAME -p $CASS_PASSWORD demo-dc1-stargate-service -e "select * from test.users;"

 email          | name          | state
----------------+---------------+-------
 john@gamil.com |    John Smith |    NC
  joe@gamil.com |     Joe Jones |    VA
   sue@help.com |       Sue Sas |    CA
    tom@yes.com | Tom and Jerry |    NV

(4 rows)
```

They could all be put in separate blocks here to be copy and pasted easily.

File: docs/content/en/install/local/_index.md [hunk 5]
```diff
@@ -1,441 +1,2316 @@
...
-Alternatively, you can download the individual charts directly from the project's [releases](https://github.com/k8ssandra/k8ssandra/releases) page.
+You'll use the extract credentials for subsequent authentication in deployed containers.
 {{% /alert %}}
 
-## Install K8ssandra
+### Verify cluster status
 
-The K8ssandra helm charts make installation a snap. You can override chart configurations during installation as necessary if you're an advanced user, or make changes after a default installation using `helm upgrade` at a later time.
+```bash
+kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -- /bin/bash
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:564]:
```suggestion
kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- nodetool -u demo-superuser -pw 3CAvGWc4mRna8tODJgeN status
```

(I made this same comment below on another similar block)

This version of the command cleans up the output a bit and matches I think more what you want, it will directly run the status command and output it, without having to go into the shell interactively.

File: docs/content/en/install/local/_index.md [hunk 6]
```diff
@@ -1,441 +1,2316 @@
...
 
-K8ssandra 1.3.0 can install the following versions of Apache Cassandra:
+**Output plus nodetool example:**
 
-* 3.11.7
-* 3.11.8
-* 3.11.9
-* 3.11.10
-* 4.0.0 (default)
+```bash
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:569]: The command change above will make your output a bit cleaner, like this:

```
Datacenter: dc1
===============
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address     Load       Tokens  Owns (effective)  Host ID                               Rack
UN  10.244.1.5  96.71 KiB  16      100.0%            4b95036b-1603-464f-bdee-b519fa28a079  default
UN  10.244.2.4  96.62 KiB  16      100.0%            ade61d9f-90f4-464c-8e18-dd3522c2bf3c  default
UN  10.244.3.4  96.7 KiB   16      100.0%            0f75a6fe-c91d-4c0e-9253-2235b6c9a206  default
```

File: docs/content/en/install/local/_index.md [hunk 7]
```diff
@@ -1,441 +1,2316 @@
...
+LAST DEPLOYED: Thu Jan 27 15:28:59 2022
+NAMESPACE: cert-manager
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+cert-manager v1.7.0 has been deployed successfully!
+```
+
+### Install K8ssandra Operator in the control-plane
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:855]: We will probably want to preface this portion with instructions for how to add the K8ssandra helm repo -- this is no different than in 1.x -- we'll use a different chart in it, but it's the same repository so the helm command to add the repo would be the same.

File: docs/content/en/install/local/_index.md [hunk 8]
```diff
@@ -1,441 +1,2316 @@
...
+
+**Output:**
+
+```bash
+false
+```
+
+### Generate and install ClientConfigs
+
+[create-clientconfig.sh](https://github.com/k8ssandra/k8ssandra-operator/blob/main/scripts/create-clientconfig.sh) lives in the k8ssandra-operator repo. It is used to configure access to remote clusters.
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:918]: This can probably be done outside of this doc here, but we'll want to overview somewhere what ClientConfigs are and how they are created and used.  They are key to the communication across clusters.

I just created this issue, we'll get someone to provide some intro documentation for the resource ASAP so you can use it:  https://github.com/k8ssandra/k8ssandra-operator/issues/427

File: docs/content/en/install/local/_index.md [hunk 9]
```diff
@@ -1,441 +1,2316 @@
...
+cassandra@k8ssandra-1-worker2:/$ cqlsh -u demo-superuser -p KT-ROFfbD-O9BzWS3Lxq
+Connected to demo at 127.0.0.1:9042
+[cqlsh 6.0.0 | Cassandra 4.0.1 | CQL spec 3.4.5 | Native protocol v5]
+Use HELP for help.
+demo-superuser@cqlsh> describe keyspaces;
+
+data_endpoint_auth  system_auth         system_schema  system_views
+system              system_distributed  system_traces  system_virtual_schema
+
+demo-superuser@cqlsh> CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'dc1' : 3, 'dc2' : 3, 'dc3': 3};
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:1362]: The comment above related to breaking these commands out into non-interactive shell commands applies here as well.

If you do indeed want to swap them out, just be sure to get the right replication strategy since they are different in the examples between the single and multi cluster examples.

File: docs/content/en/install/local/_index.md [hunk 10]
```diff
@@ -1,441 +1,2316 @@
...
+----------------+---------------+-------
+ john@gamil.com |    John Smith |    NC
+  joe@gamil.com |     Joe Jones |    VA
+   sue@help.com |       Sue Sas |    CA
+    tom@yes.com | Tom and Jerry |    NV
+
+(4 rows)
+```
+
+## Helm
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:1444]: So this helm portion here is what I mentioned above as probably wanting to have.. so we just need to figure out where we want it to go, but this is what we need.

File: docs/content/en/install/local/_index.md [hunk 11]
```diff
@@ -1,441 +1,2316 @@
...
+```console
+helm search repo k8ssandra-operator
+```
+
+```console
+NAME                                CHART VERSION   APP VERSION DESCRIPTION
+k8ssandra/k8ssandra-operator        0.32.0          1.0.0       Kubernetes operator which handles the provision...
+```
+
+### Single Cluster
```
Comment (jdonenine) [inline docs/content/en/install/local/_index.md:1470]: All the content from here down duplicates what's above... did you mean for this to be here still?

File: docs/content/en/_index.md [hunk 12]
```diff
@@ -12,44 +12,25 @@ description: "K8ssandra documentation: architecture, configuration, guided tasks
...
-The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. Check out the sections at left based on your needs and be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
+The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. 
 
-{{% alert title="Tip" color="primary" %}}
-If you're impatient, jump right in with our **[install]({{< relref "install" >}})** topics!
-{{% /alert %}}
+Be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
 
-## What is K8ssandra?
+## Features for single- and multi-cluster Kubernetes environments
```
Comment (jdonenine) [inline docs/content/en/_index.md:19]: So I think this table content is good, but where do we want this to live?

Would this go on the "common" docs set? If so this stuff should go in a separate PR targeting the `docs` branch.  Or do we want this on both versions -- basically everywhere?  I would have though it should go in the common set.

File: docs/content/en/_index.md [hunk 13]
```diff
@@ -12,44 +12,25 @@ description: "K8ssandra documentation: architecture, configuration, guided tasks
 type: docs
 ---
 
-The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. Check out the sections at left based on your needs and be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
+The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks.
```
Comment (jdonenine) [inline docs/content/en/_index.md:15]: This is probably just an idea in your head at this point... in which case let's save that convo for another day, but... What other content would you want on this intro page?  Is this where you'd want to see some intro and architecture content for operator deployed clusters?

File: docs/content/en/_index.md [hunk 14]
```diff
@@ -12,44 +12,25 @@ description: "K8ssandra documentation: architecture, configuration, guided tasks
...
-{{% alert title="Tip" color="primary" %}}
-If you're impatient, jump right in with our **[install]({{< relref "install" >}})** topics!
-{{% /alert %}}
+Be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
 
-## What is K8ssandra?
+## Features for single- and multi-cluster Kubernetes environments
 
-K8ssandra is a cloud native distribution of Apache Cassandra® (Cassandra) meant to run on Kubernetes. Accompanying Cassandra is a suite of tools to ease and automate operational tasks. This includes metrics, data anti-entropy services, and backup tooling. As part of K8ssandra's installation process all of these components are installed and wired together freeing your teams from having to perform the tedious plumbing of components.
+| K8ssandra v1.4.x      | K8ssandra Operator v1.0.0 |
```
Comment (jdonenine) [inline docs/content/en/_index.md:21]: I'm thinking we should avoid the version numbers here, try to talk about them more generically.

File: docs/content/en/_index.md [hunk 15]
```diff
@@ -12,44 +12,25 @@ description: "K8ssandra documentation: architecture, configuration, guided tasks
...
+| ----------- | ----------- |
+| K8ssandra v1.4.x is our initial implementation. It provides a set of separate Helm charts you can use to configure and deploy Apache Cassandra&reg; into a single-cluster, single-region Kubernetes environment. | K8ssandra Operator v1.0.0 is our most recent offering. It combines API, management, &amp; observability features under the control of a unified operator. Important enhancements include **multi-cluster** and **multi-region** support for Cassandra deployments in Kubernetes, which enables greater scalability and availability. Single cluster/region deployments are also supported with K8ssandra Operator.|
```
Comment (jdonenine) [inline docs/content/en/_index.md:23]: Overall this is a good intro, but I'm unsure about this part of the description: `It combines API, management, &amp; observability features`

What were you meaning with the API portion?

We might just want to be more generic here - observability is fairly minimal in the operator deployment, purposefully so.

File: docs/content/en/install/_index.md [hunk 16]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Install K8ssandra"
-linkTitle: "Install"
+title: "Install K8ssandra Operator"
```
Comment (jdonenine) [inline docs/content/en/install/_index.md:2]: My comments from the other page about what we want to call the thing we're installing will apply over here as well.

File: docs/content/en/release-notes/_index.md [hunk 17]
```diff
@@ -5,37 +5,52 @@ weight: 2
...
-**Latest release:** K8ssandra 1.4.0
+K8ssandra Operator 1.0.0 is a significant advancement of the overall K8ssandra project. K8ssandra Operator adds:
 
-Release date: 19-November-2021
+* Multi-cluster support for Kubernetes
+* Native support for multi-region/multi-DC Cassandra databases in Kubernetes
+
+As a platform architect or developer, you can declaratively manage the cluster topology and map Cassandra racks to cloud availability zones or failure domains.  
+
+K8ssandra Operator also adds a new observability service, which allows you to choose and deploy your preferred metrics collection and visualization tools. That is, with K8ssandra Operator 1.0.0, you are not limited to using only the Prometheus and Grafana deployed by K8ssandra 1.4.x.
```
Comment (jdonenine) [inline docs/content/en/release-notes/_index.md:17]: So, while this is intended to be true, we haven't actually tried any other tools yet besides prometheus, so we should probably save this for later.

File: docs/content/en/release-notes/_index.md [hunk 18]
```diff
@@ -5,37 +5,52 @@ weight: 2
...
 {{% /alert %}}
 
 ## Prerequisites
 
-* A Kubernetes environment from v1.17 (minimum supported) up to v1.22 (current tested upper bound) - local or via a supported cloud provider
-* [Helm](https://helm.sh/) v3. Note that K8ssandra 1.4.0 has been tested with Helm v3.5.3. Recommendation: specifically avoid Helm 3.6.0 and 3.6.1 due to a known CVE and subsequent regression. Our testing will resume with Helm v3.6.2 and later. Related: see GH issue [1103](https://github.com/k8ssandra/k8ssandra/issues/1103).
+* A Kubernetes environment from v1.19 (minimum supported) up to v1.22 (current tested upper bound) - local or via a supported cloud provider
+* [Helm](https://helm.sh/) v3. Note that K8ssandra 1.4.x and K8ssandra Operator have been tested with Helm v3.5.3. Recommendation: specifically avoid Helm 3.6.0 and 3.6.1 due to a known CVE and subsequent regression. Our testing may resume with Helm v3.6.2 and later. Related: see GH issue [1103](https://github.com/k8ssandra/k8ssandra/issues/1103).
```
Comment (jdonenine) [inline docs/content/en/release-notes/_index.md:38]: We don't really matrix test for helm versions since it's a client side too, but anecdotally we may want to remove this stuff, I checked my system and I'm using:

```
version.BuildInfo{Version:"v3.8.0", GitCommit:"d14138609b01886f544b2025f5000351c9eb092e", GitTreeState:"clean", GoVersion:"go1.17.6"}
```

On one system and then

```
version.BuildInfo{Version:"v3.5.2", GitCommit:"167aac70832d3a384f65f9745335e9fb40169dc2", GitTreeState:"dirty", GoVersion:"go1.15.7"}
```

On another, both work as expected.

File: docs/content/en/_index.md (hunk 15 above)
Comment (johnsmartco) [reply docs/content/en/_index.md:23]: By "API," I was thinking of Stargate.  I should probably specify the features by name: Stargate (API), Reaper (anti-entropy repairs), Medusa (backup/restore).

File: docs/content/en/release-notes/_index.md (hunk 17 above)
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md:17]: Ok, that makes sense. Will comment out or remove, for now.

File: docs/content/en/install/local/_index.md [hunk 19]
```diff
@@ -1,441 +1,2316 @@
...
 
-As K8ssandra deploys on a K8s cluster, one must be available to target for installation. The K8s environment may be a local version running on your development machine, an on-premises self-hosted environment, or a managed cloud offering.
+```bash
+cd ~/github
+git clone https://github.com/k8ssandra/k8ssandra-operator.git
+cd k8ssandra-operator
+```
 
-K8ssandra works with the following versions of Kubernetes either standalone or via a cloud provider:
+Invoke `make` with the following parameters:
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:80]: In my next commit, I briefly mention that the make... invokes the script, as an FYI for the user, and so they know it's not necessary to run the script directly. They can use use make and its target...

File: docs/content/en/install/local/_index.md [hunk 20]
```diff
@@ -1,441 +1,2316 @@
...
+* [Kustomize]({{< relref "#kustomize" >}}): Install via Kustomize - a declarative approach 
+* [Next steps]({{< relref "#next-steps" >}}): Quick start info including helper scripts
+
+## Prerequisites
+
+Make sure you have the following installed before going through this topic. 
+
+* [kind](#kind)
+* [kubectx](#kubectx)
+* [yq (YAML processor)](#yq)
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:28]: Hi @jdonenine .  Thanks for this comment.  On macOS, what's the version of `gnu-getopt` that devs should use?

[the remark this answers is not in the document]
Comment (jdonenine) [reply docs/content/en/install/local/_index.md:28]: That's a good question, I don't know if any particular version is required, but here's what I have on my system, which works:

```
% which getopt
/usr/local/opt/gnu-getopt/bin/getopt
% getopt --version
getopt from util-linux 2.37.3
```

File: docs/content/en/install/local/_index.md [hunk 21]
```diff
@@ -1,441 +1,2316 @@
...
+k8ssandra-0-worker4         Ready    <none>                 42s   v1.22.4
+```
 
-* [K3D](https://k3d.io/)
-* [Minikube](https://minikube.sigs.k8s.io/docs/start/)
-* [Kind](https://kind.sigs.k8s.io/)
+### Deploy cert-manager
 
-The instructions in this section focus on the Docker container solutions above, but the general instructions should work for other environments as well.
+Update your helm repo and set the context:
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:131]: Done in next commit.

File: docs/content/en/install/local/_index.md [hunk 22]
```diff
@@ -1,441 +1,2316 @@
...
 
 ```bash
-namespace/local-path-storage created
-serviceaccount/local-path-provisioner-service-account created
-clusterrole.rbac.authorization.k8s.io/local-path-provisioner-role created
-clusterrolebinding.rbac.authorization.k8s.io/local-path-provisioner-bind created
-deployment.apps/local-path-provisioner created
-storageclass.storage.k8s.io/local-path created
-configmap/local-path-config created
+$ kubectl get pods -n k8ssandra-operator
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:270]: Good catch - done in next commit.

File: docs/content/en/install/local/_index.md (hunk 1 above)
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:1299]: Thanks - inc'd in next commit.

File: docs/content/en/install/local/_index.md (hunk 2 above)
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:1304]: Got it, thanks. Nice follow-up, appreciated!   Will be in the next commit.

File: docs/content/en/install/local/_index.md [hunk 23]
```diff
@@ -1,441 +1,2316 @@
...
+echo $CASS_PASSWORD
+```
+
+**Output example - your value will be different:**
+```bash
+KT-ROFfbD-O9BzWS3Lxq
+```
+
+{{% alert title="Tip" color="success" %}}
+You'll use the extract credentials for subsequent authentication in deployed containers.
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:1289]: Done in next commit.

File: docs/content/en/install/local/_index.md [hunk 24]
```diff
@@ -1,441 +1,2316 @@
...
+LAST DEPLOYED: Mon Jan 31 12:30:40 2022
+NAMESPACE: k8ssandra-operator
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
 
-    **Output**:
+{{% alert title="Tip" color="success" %}}
+Optionally, you can use `--set global.clusterScoped=true` to install K8ssandra Operator cluster-scoped:
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:201]: Done in next commit.

File: docs/content/en/install/local/_index.md (hunk 3 above)
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:558]: Done in next commit.

File: docs/content/en/install/local/_index.md (hunk 4 above)
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:586]: Great idea - easier for user.  Done in the next commit.

File: docs/content/en/install/local/_index.md (hunk 5 above)
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:564]: Yes, but the -pw value should be `ACK7dO9qpsghIme-wvfI` from the result just above it.   So altogether, we have:


### Verify cluster status

```bash
kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -c cassandra -- nodetool -u demo-superuser -pw ACK7dO9qpsghIme-wvfI status
```

**Output:**

```bash
Defaulted container "cassandra" out of: cassandra, server-system-logger, jmx-credentials (init), server-config-init (init)
Datacenter: dc1
===============
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address     Load       Tokens  Owns (effective)  Host ID                               Rack
UN  10.244.4.4  96.71 KiB  16      100.0%            bbdb6a9a-063b-4565-9704-f4caa6fd80f1  default
UN  10.244.2.5  96.72 KiB  16      100.0%            61dfa8cc-2a8b-4e8f-ae82-01c51833e0ba  default
UN  10.244.3.4  96.63 KiB  16      100.0%            369aa179-d96e-4f21-a893-f6e6dc84b396  default
```

File: docs/content/en/install/local/_index.md (hunk 6 above)
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:569]: Thanks! In next commit.

File: docs/content/en/install/local/_index.md (hunk 7 above)
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:855]: Ok, something like this? 

First, configure the K8ssandra Helm repository:

```console
helm repo add k8ssandra https://helm.k8ssandra.io/stable
```

Update your Helm repository cache:

```console
helm repo update
```

Verify that you see the `k8ssandra-operator` chart:

```console
helm search repo k8ssandra-operator
```

File: docs/content/en/install/local/_index.md (hunk 8 above)
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:918]: Ok I've noted that separate doc need. Will resolve this comment, for now.

File: docs/content/en/install/local/_index.md (hunk 11 above)
Comment (johnsmartco) [reply docs/content/en/install/local/_index.md:1470]: Oh ok, I see what you mean.  In my next commit, I'll remove the whole ## helm section, but grab the first portion and insert it in the place that you noted in comments above.

File: docs/content/en/_index.md (hunk 12 above)
Comment (johnsmartco) [reply docs/content/en/_index.md:19]: Yes, after this commit, I will do surgery to get this into the common (docs branch) content.

File: docs/content/en/_index.md (hunk 13 above)
Comment (johnsmartco) [reply docs/content/en/_index.md:15]: Yes, more common intro material will go into this page.

File: docs/content/en/_index.md (hunk 15 above)
Comment (johnsmartco) [reply docs/content/en/_index.md]: Updated in most recent commit. Note that right after this merge (if approved), I will move the top-level doc home _index.md over to the common `docs` branch.  Right, it's not for the docs-v2 branch.

File: docs/content/en/release-notes/_index.md [hunk 25]
```diff
@@ -5,37 +5,52 @@ weight: 2
 description: Release notes for the open-source K8ssandra community project.
 ---
 
-K8ssandra provides a production-ready platform for running Apache Cassandra&reg; on Kubernetes. This includes automation for operational tasks such as repairs, backup and restores, and monitoring. Also deployed is Stargate, an open source data gateway that lets you interact programmatically with your Kubernetes-hosted Cassandra resources via a well-defined API. 
+K8ssandra and K8ssandra Operator provide production-ready platform for running Apache Cassandra&reg; on Kubernetes. This includes automation for operational tasks such as repairs, backup and restores, and monitoring. Also deployed is Stargate, an open source data gateway that lets you interact programmatically with your Kubernetes-hosted Cassandra resources via a well-defined API.
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md:8]: Right, I will move it to the common docs branch.

File: docs/content/en/_index.md [hunk 26]
```diff
@@ -18,19 +18,17 @@ Be sure to leave us a <a class="github-button" href="https://github.com/k8ssandr
...
-| Start in the K8ssandra v1.4.x [install](https://docs-staging-v1.k8ssandra.io/install/local/) topics, which include the steps for single-cluster installs on local or cloud-provider Kubernetes platforms. | Start in the K8ssandra Operator v1.0.0 [local install]({{< relref "install/local" >}}) topic, which focuses on multi-cluster or single-cluster installs on local dev  **kind** Kubernetes, using helper scripts, `helm`, or `kustomize`.
+| K8ssandra v1.x was our initial implementation. It provides a set of separate Helm charts you can use to configure and deploy Apache Cassandra&reg; into a single-cluster, single-region Kubernetes environment. | K8ssandra Operator v1.0.0 is our most recent offering. It combines API, management, &amp; observability features under the control of a unified operator. Important enhancements include **multi-cluster** and **multi-region** support for Cassandra deployments in Kubernetes, which enables greater scalability and availability. Single cluster/region deployments are also supported with K8ssandra Operator.|
```
Comment (jdonenine) [inline docs/content/en/_index.md]: This is minor, but I think you can omit the version numbers for the operator here so that it can be more general and stand the test of time.

File: docs/content/en/release-notes/_index.md (hunk 18 above)
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md:38]: Updated in next commit.

File: docs/content/en/_index.md [hunk 27]
```diff
@@ -18,19 +18,17 @@ Be sure to leave us a <a class="github-button" href="https://github.com/k8ssandr
...
+| Start in the K8ssandra v1.4.x [install](https://docs-staging-v1.k8ssandra.io/install/local/) topics, which include the steps for single-cluster installs on local or cloud-provider Kubernetes platforms. | For enhanced capabilities, start in the K8ssandra Operator v1.0.0 [local install]({{< relref "install/local" >}}) topic, which focuses on single- or multi-cluster deployments on local dev  **kind** Kubernetes clusters, using the provided `make` commands, `helm`, or `kustomize`.
```
Comment (jdonenine) [inline docs/content/en/_index.md:24]: This is a question that I grappled with writing the release blog... should we be leading with K8ssandra Operator as THE way from here on out, trying to actively push people to using it over the original charts.

Obviously that's up to each users opinion in terms of stability and functionality, but I'm leaning towards, let's start leading with the operator as the way (insert Mandelorian emoji here 😉 ).

With that in mind should we perhaps reword this a bit to introduce K8ssandra Operator first and also just present it as the default - and not just the "for advanced cases" sort of solution?

**Review by jdonenine — APPROVED** [review]
Thanks for the updates, looks good.  I left a question in there to think about, but we can address that separately if you want to get this moving.

File: docs/content/en/install/_index.md (hunk 16 above)
Comment (johnsmartco) [reply docs/content/en/install/_index.md:2]: Just did a commit to fix this.

File: docs/content/en/release-notes/_index.md (hunk 17 above)
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md]: Yeah, I minimized the observability text.

File: docs/content/en/_index.md (hunk 26 above)
Comment (johnsmartco) [reply docs/content/en/_index.md]: Reworded...

File: docs/content/en/_index.md (hunk 27 above)
Comment (johnsmartco) [reply docs/content/en/_index.md:24]: Yeah let me go with that (ha, Mandelorian), K8ssandra Operator as the recommended Kubernetes + Cassandra solution.
